tests/integration/features/chat-4/scheduled-messages.feature

```diff
@@ -0,0 +1,111 @@
+Feature: chat-4/scheduling
+  Background:
+    Given user "participant1" exists
+    Given user "participant2" exists
+    Given user "participant1" creates room "room" (v4)
+      | roomType | 2    |
+      | roomName | room |
+    And user "participant1" adds user "participant2" to room "room" with 200 (v4)
+    And user "participant2" sends message "Message" to room "room" with 201
+
+  Scenario: Schedule a message
+    When user "participant1" schedules a message to room "room" with 201
+      | message | Message 1  |
+      | sendAt  | 1985514582 |
+    Then user "participant1" sees the following scheduled messages in room "room" with 200
+      | id        | roomId | actorType | actorId      | threadId | parentId | message   | messageType | sendAt      | metaData                                         |
+      | Message 1 | room   | users     | participant1 | 0        | null     | Message 1 | comment     | 1985514582  | {"thread_title":"","silent":false,"thread_id":0} |
```

Comment:
metaData is camelCase in chat messages, so should also be here?
https://github.com/nextcloud/spreed/blob/9112cc90fd85b1ee7066eafb32cb8c3a1c4f4196/lib/ResponseDefinitions.php#L107-L118